### ISSUE TITLE
[Market] NullPointerException in XmppManager.send

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#192]
```
Reproduced twice on 2.2.4

java.lang.NullPointerException
at com.googlecode.gtalksms.XmppManager.send(XmppManager.java:725)
at com.googlecode.gtalksms.MainService.onHandleIntent(MainService.java:219)
at 
com.googlecode.gtalksms.MainService$ServiceHandler.handleMessage(MainService.jav
a:126)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:143)
at android.os.HandlerThread.run(HandlerThread.java:60)
```

Original issue reported on code.google.com by `Florent....@gmail.com` on 24 Jun 2011 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
http://code.google.com/p/gtalksms/source/browse/src/com/googlecode/gtalksms/Xmpp
Manager.java?name=f7973ae6160b#725

Should be easy to fix. At some point I am gonna have to find out what sets the 
XMPPConnection to null.
```

Original comment by `fschm...@gmail.com` on 24 Jun 2011 at 1:21
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v3.0
```

Original comment by `Florent....@gmail.com` on 15 Jul 2011 at 3:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
